### DISCUSSION TITLE
example plot_hamersley.py failing

### DISCUSSION THREAD

**Discussion by ShebMichel** [Loop3D/map2loop#31] [General]
Here is the error message:

2024-02-06 16:05:21 Unused keys from legacy format ['msc', 'msn', 'mst', 'mtc', 'mscm', 'mcom', 'minf', 'fdipdir_flag', 'deposit_dist']
2024-02-06 16:05:26 Warning 1: PROJ: proj_create_from_database: Open of /opt/conda/share/proj failed
2024-02-06 16:05:26 Warp failed for DTM

**Comment by lachlangrose** [comment]
Hi @ShebMichel,

thats not an issue with map2loop, that is an issue with your environment. You may need to reinstall geopandas/pyproj... unfortunately gdal dependencies are difficult to manage. I'm going to convert it into a discussion topic.

@ayLa-e did you find a solution to this or was a new environment the solution?

**Comment by lachlangrose** [comment]
@ShebMichel 

this is a message from @ayLa-e 

> Hey @lachlangrose ,
I believe it was a gdal issue. A new environment was the way to go.
@ShebMichel
The initial part of the error message that's been copied in about unused legacy keys is just the expected output of map2loop when using a legacy configuration file (hjson file). If you use the legacy variable names (ie the variable names used in map2loop version 2), the inbuilt converter will output a list of any unused variable names as shown in your example above. Once you get your environment working correctly, you should still get the output line about unused keys.  Hope this makes sense. Please let me know if you'd like more clarification
